games-fps/yamagi-quake2/yamagi-quake2-8.60.ebuild

```diff
@@ -40,7 +41,7 @@ RDEPEND="
 		)
 		openal? ( media-libs/openal )
 		!openal? ( media-libs/libsdl2[sound] )
-		opengl? ( media-libs/libglvnd[X] )
+		opengl? ( media-libs/libglvnd )
```

Comment:
I don't think this is right? From a quick grep I see that it uses libGL and not libOpenGL (without USE=X, libGL.so does *not* exist, often `-lGL` can be dropped-in for `-lOpenGL` if it does not use GLX-specific functions though).
```make
release/ref_gl1.so : LDLIBS += -lGL
```

Reply:
Yes, thank you.